homeassistant/brands/ikea.json

```diff
@@ -0,0 +1,6 @@
+{
+  "domain": "ikea",
+  "name": "IKEA",
+  "integrations": ["tradfri"],
+  "iot_standards": ["zigbee"]
```

Comment:
We have a similar thing with `Hue`, which we also didn't apply yet.

The reason is, that "Adding a Zigbee device" could be confused with "adding it to the Tradfri hub".

Until that is solved, I suggest not to go this route.

Reply:
Ahh, I see. Makes sense. Do you have a link to where this is being discussed? So I cant out it in the PR for reference.

Reply:
>  Do you have a link to where this is being discussed? 

Nope, sorry.